pandas/core/indexers/objects.py

```diff
@@ -93,7 +93,6 @@ def get_window_bounds(
 
         end = np.clip(end, 0, num_values)
         start = np.clip(start, 0, num_values)
-
```

Comment:
Do you mind reverting these whitespace changes? It's somewhat distracting from the other changes in this PR

Reply:
Sure.